src/pages/authorization/agent-authorization/index.mdx

```diff
@@ -0,0 +1,676 @@
+| `access_token`    | A token you can use to call LiveChat APIs on behalf of the user.                                                                                                                                                                           |
+| `account_id`      | The ID of the Agent's account                                                                                                                                                                                                              |
+| `client_id`       | **Client Id** of your app                                                                                                                                                                                                                  |
+| `expires_in`      | A number in seconds specifying how long the `access_token` will be valid. When it expires, you will need to generate new `access_token` using `refresh_token` (read [Using the refresh token](#using-the-refresh-token) for more details). |
+| `organization_id` | The ID of the organization's account.                                                                                                                                                                                                      |
+| `scope`           | A comma-separated list of permissions an `access_token` has.                                                                                                                                                                               |
+| `refresh_token`   | A token that can be used to generate new access tokens. Returned optionally.                                                                                                                                                               |
+| `token_type`      | Value: `Bearer`                                                                                                                                                                                                                            |
+
+💡 **Got stuck? See common** [**Errors**](#errors).
+
+</Text>
+<Code>
+<CodeSample path={'VALIDATE AN ACCESS TOKEN'}>
+
+```shell
+curl "https://accounts.livechat.com/v2/info"
+  -H "Authorization: Bearer <access_token>"
+```
+
+</CodeSample>
+<CodeResponse title={'response'}>
+
+```json
+{
+    "access_token": "dal:test_YTJQ6GDVgQf8kQDPw",
+    "account_id": "bbe8b147-d60e-46ac-a1e5-1e94b11ea6e1",
+    "client_id": "9cbf3a968289727cb3cdfe83ab1d9836",
+    "expires_in": 28725,
+    "organization_id": "390e44e6-f1e6-0368c-z6ddb-74g14508c2ex",
+    "scope": "chats--all:ro,chats--all:rw",
+    "token_type": "Bearer"
+}
+```
+
+</CodeResponse>
+</Code>
+</Section>
```

Comment:
# Errors
`<link to /authorization/authorizing-api-calls/#errors>`

Reply:
We left the Errors section in the Authorizing API calls, which is not optimal, as errors should be close to methods. But we would have to duplicate the content - paste it into the Agent authorization and Customer authorization docs, which isn't optimal either :/ 

I propose to add the Errors section in both Agent authorization and Customer authorization and link to Erros in the Authorizing API calls document.

Reply:
Okay, I agree that seems most reasonable